updater/lib/dependabot/file_fetcher_command.rb

```diff
@@ -120,9 +120,9 @@ def dependency_files_for_globs
 
           has_glob = true
           dir = dir.delete_prefix("/")
-          Dir.glob(dir, File::FNM_DOTMATCH).select { |d| File.directory?(d) }
+          Dir.glob(dir, File::FNM_DOTMATCH).select { |d| File.directory?(d) }.map { |d| "/#{d}" }
```

Comment:
why prepend a `/` ? is that just what the updater expects?

Reply:
The API sends all directories with a starting `/` (at least it tries to) so we need to match that in order to properly uniq the directories.